spec/controllers/network_topology_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe NetworkTopologyController do
+  render_views
+  before(:each) do
+    set_user_privileges
+  end
+
+  it "renders index" do
+    get :index
+    expect(response.status).to eq(302)
+    expect(response).to redirect_to(:action => 'show')
+  end
+
+  it "renders show screen per provider id" do
+    EvmSpecHelper.create_guid_miq_server_zone
+    ems = FactoryGirl.create(:ems_openstack)
```

Comment:
the network provider is not an independent one? it needs to be created via other (in this case openstack) provider?